tendrl/commons/utils/alert_utils.py

```diff
@@ -0,0 +1,40 @@
+import json
+import os
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils.time_utils import now as tendrl_now
+
+
+def alert_job_status(curr_value, msg, integration_id=None, cluster_name=None):
+    alert = {}
+    alert['source'] = NS.publisher_id
+    alert['classification'] = 'cluster'
+    alert['pid'] = os.getpid()
+    alert['time_stamp'] = tendrl_now().isoformat()
+    alert['alert_type'] = 'STATUS'
+    severity = "INFO"
+    if curr_value.lower() == "failed":
+        severity = "WARNING"
+    alert['severity'] = severity
+    alert['resource'] = 'cluster'
```

Comment:
In resource, you have to give like job status or like volume status based on the alert type

Reply:
in future, if we allow the user choose what are all the alert types they want to raise the alert then we can restrict are allow the tendrl by using this field only.  These resource fields act as alert types in etcd.

under utilization alert types : brick utilization, cpu utilization ..etc these will come. so resource are the sub alert types under alert types

Reply:
I think I tried with `job_status` but faced some issue. Anyway will try once and change if required.